Project.toml

```diff
@@ -3,6 +3,7 @@ uuid = "0bf59076-c3b1-5ca4-86bd-e02cd72cde3d"
 version = "0.2.28"
```

Comment:
Please also increment the version here.

Reply:
Should we make a minor version bump? Seems like quite a significant change given the introduction of `HMCKernel` too

Reply:
Yeah a minor version bump sounds suitable.